app.js

```diff
@@ -13,8 +14,10 @@ const session            = require('express-session');
 const MongoStore         = require('connect-mongo')(session);
 const mongoose           = require('mongoose');
 const flash              = require('connect-flash');
+const multer             = require('multer');
+const upload             = multer({ dest: './public/uploads/' });
 
-mongoose.connect('mongodb://localhost:27017/tumblr-lab-development');
+mongoose.connect(process.env.MONGODB_URI);
```

Comment:
This means the app won't work locally. You could do something like 

mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost:27017/tumblr-lab-development');

Then if it doesn't find an environment variable it will fall back to the local server for development

Reply:
Don't worry though, we didn't really cover this yet